### PR TITLE
STO 1.67.1 release notes DOC-3657

### PR DESCRIPTION
**Pull request by douglas-j-bothwell**
# Harness Developer Pull Request

Preview links: 
- Update STO relnotes: [Latest: Version 1.67.1](https://64efd44874548a02c485fcb7--harness-developer.netlify.app/release-notes/security-testing-orchestration#latest-version-1671)
- Update Settings and Additional CLI arguments fields in BDH scanner reference:  [Settings](https://64efd44874548a02c485fcb7--harness-developer.netlify.app/docs/security-testing-orchestration/sto-techref-category/black-duck-hub-scanner-reference#settings)
- What's new [Latest - August 30, 2023](https://64efd44874548a02c485fcb7--harness-developer.netlify.app/release-notes/whats-new#latest---august-30-2023)

### CODE REVIEW

File: release-notes/security-testing-orchestration.md [hunk 1]
```diff
@@ -19,51 +19,82 @@ Review the notes below for details about recent changes to Security Testing Orch
...
+    ![](static/sto-exemption-details-no-baseline-selected.png)
+
+  These enhancements are behind the Feature Flag `FF_STO_EXEMPTION_DETAILS`. Contact [Harness Support](mailto:support@harness.io) to enable the feature. (STO-5056)
+
+-->
+
 ### Fixed issues
 
-This update to the SonarQube runner includes the following fixed issues:
+- Fixed a pagination issue in the Security Tests page where switching between different pages &mdash; for example, from issues 1-20 to issues 21-40 &mdash; resulted in a "Failed to get issues" error. (STO-6465)
```
Comment (aklaft) [inline release-notes/security-testing-orchestration.md:49]: the error happened when you were looking at a page (21-40) and switched to another issue for which those settings were invalid

File: release-notes/security-testing-orchestration.md [hunk 2]
```diff
@@ -19,51 +19,82 @@ Review the notes below for details about recent changes to Security Testing Orch
...
+
+- Fixed an issue where the Security Tests page would fail with a 500 internal error if the scan detected no issues. (STO-6437, ZD-49803)
+
+- Fixed an issue where the Black Duck Hub step ran DOCKER scans only by default. With this fix, the scanner runs DETECTOR and SIGNATURE scans by default as well. (STO-6447)
```
Comment (aklaft) [inline release-notes/security-testing-orchestration.md:53]: not sure if was an "issue" per se (as far as customers knew) , we're just choosing to run more scans now

File: release-notes/security-testing-orchestration.md (hunk 1 above)
Comment (douglas-j-bothwell) [reply release-notes/security-testing-orchestration.md:49]: @aklaft Revised as follows:
- Fixed a pagination issue in the Security Tests page where switching between different pages resulted in a "Failed to get issues" error. The error occurred when switching from a page of issues (such as 21-40) to another issue for which those settings were invalid. (STO-6465)

File: release-notes/security-testing-orchestration.md (hunk 2 above)
Comment (douglas-j-bothwell) [reply release-notes/security-testing-orchestration.md:53]: @aklaft Revised as follows:
- Previously, the Black Duck Hub step ran DOCKER scans only by default. With this release, the scanner runs DETECTOR and SIGNATURE scans by default as well. (STO-6447)

File: release-notes/security-testing-orchestration.md [hunk 3]
```diff
@@ -19,51 +19,82 @@ Review the notes below for details about recent changes to Security Testing Orch
...
+-->
+
 ### Fixed issues
 
-This update to the SonarQube runner includes the following fixed issues:
+- Fixed a pagination issue in the Security Tests page where switching between different pages resulted in a "Failed to get issues" error. The error occurred when switching from a page of issues (such as 21-40) to another issue for which those settings were invalid. (STO-6465)
+
+- Fixed an issue where the Security Tests page would fail with a 500 internal error if the scan detected no issues. (STO-6437, ZD-49803)
+
+- Previously, the Black Duck Hub step ran DOCKER scans only by default. With this release, the scanner runs DETECTOR and SIGNATURE scans by default as well. (STO-6447)
```
Comment (brian-f-harness) [inline release-notes/security-testing-orchestration.md:53]: assume the capitalization is intentional here

**Review by brian-f-harness — APPROVED** [review]
LGTM

File: release-notes/security-testing-orchestration.md (hunk 3 above)
Comment (douglas-j-bothwell) [reply release-notes/security-testing-orchestration.md:53]: Yup!

**Pull request merged**